src/Campaigns/resources/types.ts

```diff
@@ -45,6 +45,7 @@ export type GiveCampaignOptions = {
         showCampaignListTableNotice: boolean
         showCampaignFormNotice: boolean
         showCampaignSettingsNotice: boolean
+        orphanedForms?: []
```

Comment:
Is this being used?

Reply:
It's a leftover, I'll remove. Good catch